readme.md

```diff
@@ -102,24 +99,22 @@ mbo.finalize().then(function(mediaBlobs) {
 ```
 
 ### Split Method
-The split method allows the author to split a *blob* into two separate MediaBlobs at a given time.
+The split method allows the author to split a *blob* into two separate MediaBlobs at a given time. Due to the nature of this operation, it should be the last
```

Comment:
Does MediaBlob support calling split() multiple times to produce more than two separate MediaBlobs?

Reply:
split() operation can be batched multiple times on the MediaBlobOperation but we will throw an error on finalize().
Ex:
```
mbo.split(5000);
mbo.trim(0,3000);
mbo.split(1500);
mbo.finalize();  // error because mbo.split(5000) is not the last operation before calling finalize()
```

Reply:
What about two splits at the end of the operation list like:

`
mbo.trim(0,3000);
mbo.split(1500);
mbo.split(5000);
mbo.finalize()
`

I assume this will also create an error?

Reply:
Yes. Since `mbo.split(1500);` is not the last operation here.